### PR TITLE
Update Install_AWSParallelCluster.md

### PR DESCRIPTION
**Pull request by dchesterNIH**
### Description

*Provide a description of your changes here. If this PR is related to an issue, list the issue number/name here*

### Assignee

*Assignees: @kyleoconnell-NIH and @petersonjdNIH*

## PR checklist
*Please ensure the following:*
- [ ] This comment contains a description of changes (with reason).
- [ ] All changes were tested.
- [ ] If you've fixed a bug mention the issue number/name.
- [ ] Apply approriate tags (e.g. documentation, bug)

### CODE REVIEW

**Comment by dchesterNIH** [comment]
updated AWS parrallel cluster with UI instructions

File: docs/Install_AWSParallelCluster.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-# Ways to install AWS ParrallelCluster:
+#Installing the AWS ParrallelCluster User interface (UI)
+
+ ## To install and instance of the AWS Parrallel Cluster UI choose an AWS Cloudformation quick-create link for the AWS region you want cluster in.
```
Comment (kyleoconnell-NIH) [inline docs/Install_AWSParallelCluster.md]: You want THE cluster in

File: docs/Install_AWSParallelCluster.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
-# Ways to install AWS ParrallelCluster:
+#Installing the AWS ParrallelCluster User interface (UI)
+
+ ## To install and instance of the AWS Parrallel Cluster UI choose an AWS Cloudformation quick-create link for the AWS region you want cluster in.
+ 
+  Follow the instructions [here](https://docs.aws.amazon.com/parallelcluster/latest/ug/install-pcui-v3.html)
+ 
+#Installing the AWS ParrallelCluster CLI:
```
Comment (kyleoconnell-NIH) [inline docs/Install_AWSParallelCluster.md]: Typo Parallel

**Comment by kyleoconnell-NIH** [comment]
Where is the UI part? I expected screen shots.

**Review by kyleoconnell-NIH — APPROVED** [review]

**Pull request merged**